src/harvest-rpde.js

```diff
@@ -0,0 +1,205 @@
+const { default: axios } = require('axios');
+const { performance } = require('perf_hooks');
+const { FeedPageChecker } = require('@openactive/rpde-validator');
+const sleep = require('util').promisify(setTimeout);
+
+const { createFeedContext, progressFromContext } = require('./feed-context-utils');
+const { millisToMinutesAndSeconds } = require('./utils');
+
+/**
+ * @typedef {import('../src/models/core').FeedContext} FeedContext
```

Comment:
Unfortunately (unless something's improved since earlier?) these types won't be accessible to client libraries. In order to do that, you need to use `tsc` to generate `.d.ts` files. The `packages/test-interface-criteria` project in Test Suite is an example of a project that does this

Reply:
Oh wait I've just seen that the tsconfig.json should generate `.d.ts` files. Do you just need to run `npx tsc` in order to generate them?

Reply:
`dist` was in the gitignore, the generated files were in there, have now added them